src/lib/Microsoft.Fx.Portability.Cci/Analyzer/DependencyFinderEngine.cs

```diff
@@ -60,22 +60,30 @@ private void FindDependencies(IProgressTask progressTask)
                 {
                     foreach (var dep in GetDependencies(filename))
                     {
-                        MemberInfo m = new MemberInfo() { MemberDocId = dep.MemberDocId, TypeDocId = dep.TypeDocId, DefinedInAssemblyIdentity = dep.DefinedInAssemblyIdentity };
+                        var m = new MemberInfo
+                        {
+                            MemberDocId = dep.MemberDocId,
+                            TypeDocId = dep.TypeDocId,
+                            DefinedInAssemblyIdentity = dep.DefinedInAssemblyIdentity
+                        };
 
                         // Add this memberinfo
                         var newassembly = new HashSet<AssemblyInfo>
                         {
                             dep.CallingAssembly
                         };
```

Comment:
Consider using the overload of AddOrUpdate that takes a factory for the initial value as well, to avoid allocating the HashSet whether you use it or not.

Reply:
I would prefer to take care of these in another pass.  Open https://github.com/microsoft/dotnet-apiport/issues/810 to track the issue.